### PR TITLE
PR1 functions

### PR DESCRIPTION
**Pull request by ArcGate-Akshi**
pr for functions in python

### CODE REVIEW

File: functions/calculator.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+def cal(a,b):
+   return(a+b ,a-b, a*b, a//b)
+
+add, sub, multi, divi = cal(10, 5)
+print(add, sub , multi, divi)
```
Comment (Arcgate-krishna) [inline functions/calculator.py]: Add extra line space in the end of file

File: functions/function_into_function.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+def outer(a,b):
+    def inner(a,b):
+        return a+b
+    
+    add=inner(a,b)
+    return add + 5    
+
+x = int(input("enter a: "))
+y = int(input("enter b: "))
+print(outer(x,y))
```
Comment (Arcgate-krishna) [inline functions/function_into_function.py]: Add extra line space in the end of file

File: functions/function_prac1.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+#     return num%2==0
+
+# print(is_even(1234))
+   
+##function without parameters
+def reminder():
+    return "smile"
+
+
+print(reminder())
```
Comment (Arcgate-krishna) [inline functions/function_prac1.py]: Add extra line space in the end of file

File: functions/palindrome.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+def is_palindrome(str):
+   return str == str[::-1]
+#rev = str[-1::-1]
+    #if str == str[::-1]:
+        #return True
+    #return False
+
+
+string = input("enter the string you want to check:  ")
+print(is_palindrome(string))
```
Comment (Arcgate-krishna) [inline functions/palindrome.py]: Add extra line space in the end of file

File: functions/recursive_func.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+#func that calls itself again and again
+def rec(n):
+    if n:
+       return n + rec(n-1)
+    else:
+        return 0
+    
+result = rec(10)
+print(result)
+
```
Comment (Arcgate-krishna) [inline functions/recursive_func.py:9]: remove extra line space in the end of file

File: functions/calculator.py (hunk 1 above)
Comment (ArcGate-Akshi) [reply functions/calculator.py]: I have added new line space at the end of all files in this PR

Comment (ArcGate-Akshi) [reply functions/calculator.py]: I have added new line space at the end of all files in this PR @Arcgate-krishna

**Review by Arcgate-krishna — APPROVED** [review]
LGMT

**Pull request merged**